templates/rapidsai/partials/clone_and_build_blazing.j2

```diff
@@ -0,0 +1,39 @@
+{# This partial clones the BlazingSQL repo and build and installs it into the rapids conda env. #}
+
+{# Install build prerequisites #}
+RUN gpuci_conda_retry install -y -n rapids \
```

Comment:
This install will require the blazingsql-nightly channel, should I replace it in condarc instead?

Reply:
I'd check with Mike on this.